src/templates/base.html

```diff
@@ -79,20 +79,21 @@
         background-color: #484d4d;
       }
       form input[type="submit"] {
-
+        margin: 1em;
+      }
+      form input a {
+        margin: 2em;
       }
-
 
     </style>
   </head>
   <body>
     <div class="container">
     <div id="navbar">
       <ul>
-    	<li><a href="http://localhost:8000/about">About</a></li>
-    	<li><a href="http://localhost:8000">Active links</a></li>
-    	<li><a href="http://localhost:8000/my_links">My links</a></li>
+    	<li><a href="http://localhost:8000/links/about">About</a></li>
+    	<li><a href="http://localhost:8000/links/all">Active links</a></li>
+    	<li><a href="http://localhost:8000/links/me">My links</a></li>
```

Comment:
I think it would be more future-proof to use only relative links.
```suggestion
    	<li><a href="/links/about">About</a></li>
    	<li><a href="/links/all">Active links</a></li>
    	<li><a href="/links/me">My links</a></li>
```

(Same goes for all the links in the application.)

Reply:
Changed